server/athenian/api/models/state/models.py

```diff
@@ -640,3 +640,47 @@ class DashboardChartGroupBy(create_time_mixin(created_at=True, updated_at=True),
     DashboardChartGroupBy.jira_issue_types,
     DashboardChartGroupBy.jira_labels,
 )
+
+
+class InstallationProgress(create_time_mixin(created_at=False, updated_at=False), Base):
+    """Tracking installation progress for github accounts"""
+
+    __tablename__ = "installation_progress"
+
+    account_id = Column(
```

Comment:
Could you please format every field in one line for better readability and consistency with the surrounding models :pray: E.g.
```suggestion
    account_id = Column(Integer(), primary_key=True)
```

Also, note that `primary_key=True` implies `nullable=False` under the hood.